docs/tutorials/kubernetes/ovhcloud.md

```diff
@@ -0,0 +1,452 @@
+# Scan for vulnerabilities and misconfigurations of your OVHcloud Managed Kubernetes with Trivy
```

Comment:
Are we sure this is a "first party" tutorial and not "third party"? do we want to maintain it?

Reply:
No, we can't. The first party tutorial should be

- GitHub Actions
  - We have an official action
- CircleCI
- Travis CI
- GitLab CI
  - GitLab folks maintain our tutorial
- Bitbucket Pipelines
  - We have [an official repo](https://github.com/aquasecurity/trivy-pipe)
- (AWS CodePipeline)
  - It should be moved to third party as the blog is written by AWS, not us.
- AWS Security Hub
  - as we maintain [asff.tpl](https://aquasecurity.github.io/trivy/v0.30.4/docs/integrations/aws-security-hub/)

Reply:
Hmm in addition to those we need tutorials such as Deploy the `Trivy Operator through GitOps` Where do you see those?